classes/fields/number.php

```diff
@@ -303,7 +305,7 @@ public function pre_save( $value, $id = null, $name = null, $options = null, $fi
 	 */
 	public function format( $value = null, $name = null, $options = null, $pod = null, $id = null ) {
 
-		if ( null === $value ) {
+		if ( $this->is_empty( $value ) ) {
```

Comment:
`is_empty()` will catch `0` and `0.00` values too, not sure we want that applied here.

Reply:
`0` and `0.00` will not be seen as numeric values:
https://github.com/pods-framework/pods/blob/feature/5539-number-empty/classes/fields/number.php#L167-L177